src/components/Mandal/view/utils.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import MandalBox from "./MandalBox";
+
+export const showBoxes = (data, handleOnClick) => {
```

Comment:
```suggestion
export const showBoxes = (data, onClick) => {
```

또는

```suggestion
export const showBoxes = (data, handleClick) => {
```

이렇게 둘 중 하나로 가도 괜찮지 않을까요!